### PR TITLE
add instructions for kebechet

### LINKED ISSUE #229: `use-bots.md` should include instructions for setting up Kebechet bot [meteor]
`use-bots.md` does not provide users with instructions on how to include Kebechet bot in their repo. I suggest that we include the instructions from [here](https://github.com/AICoE/aicoe-ci/blob/master/docs/thoth-bots-setup.md#instructions-to-setup-bots-and-ci-services) to this page. 

Acceptance Critreria:

- [ ] add [instructions](https://github.com/AICoE/aicoe-ci/blob/master/docs/thoth-bots-setup.md#instructions-to-setup-bots-and-ci-services) for setting up kebechet to 'use-bots.md'

### PR DESCRIPTION
**Pull request by isabelizimm**
## Related Issues and Dependencies

closes #229 

## This introduces a breaking change

- [ ] Yes
- [X] No

## This Pull Request implements

- add details to configure thoth bots

### CODE REVIEW

**Review by oindrillac — COMMENTED** [review]
/lgtm

**Review by Shreyanand — COMMENTED** [review]
/lgtm

File: docs/source/use-bots.md [hunk 1]
```diff
@@ -8,6 +8,36 @@ An example of the Kebechet bot in action can be seen below.
...
+
+Once the application is installed, you will need to add Thoth's bots as collaborators. Navigate to your fork of `elyra-aidevsecops-tutorial`. Under the repository's **Settings**, go to **Manage Access** and click on "Invite a collaborator" and add Sesheta. Sesheta is a friendly Thoth bot who is used to help automate tasks. You might have already done this step when [building an image](build-images.md), in which case you do not have to invite Sesheta again. Please note: there is sometimes a delay in Sesheta's invite acceptance. Please note: Sesheta's invite acceptance is not instantaneous. [Follow this link](https://github.com/AICoE/aicoe-ci/issues/new?assignees=goern%2Charshad16&labels=area%2Fcyborgs%2Cbot%2Csig%2Fcyborgs&template=request_sesheta.yaml&title=Help+with+Sesheta+invite) and fill out the form to have Sesheta accept your invitation.
```
Comment (chauhankaranraj) [inline docs/source/use-bots.md:15]:
```suggestion
Once the application is installed, you will need to add Thoth's bots as collaborators. Navigate to your fork of `elyra-aidevsecops-tutorial`. Under the repository's **Settings**, go to **Manage Access** and click on "Invite a collaborator" and add Sesheta. Sesheta is a friendly Thoth bot who is used to help automate tasks. You might have already done this step when [building an image](build-images.md), in which case you do not have to invite Sesheta again. Please note: currently Sesheta does not accept invites instantaneously and automatically (see [this issue](https://github.com/AICoE/aicoe-ci/issues/126) for more details). [Follow this link](https://github.com/AICoE/aicoe-ci/issues/new?assignees=goern%2Charshad16&labels=area%2Fcyborgs%2Cbot%2Csig%2Fcyborgs&template=request_sesheta.yaml&title=Help+with+Sesheta+invite) and fill out the form to have Sesheta accept your invitation.
```
Could we explicitly mention that invite won't be accepted automatically?

File: docs/source/use-bots.md [hunk 2]
```diff
@@ -8,6 +8,36 @@ An example of the Kebechet bot in action can be seen below.
...
+
+<div style="text-align:center">
+<img alt="Invite Sesheta" src="https://raw.githubusercontent.com/aicoe/elyra-aidevsecops-tutorial/master/docs/images/InviteSesheta.png">
```
Comment (chauhankaranraj) [inline docs/source/use-bots.md:18]: Seems like the image here does not render (maybe coz `InviteSesheta.png` is not in the repo?). Is that expected?

**Review by chauhankaranraj — CHANGES_REQUESTED** [review]
small suggestion but otherwise looks good!

File: docs/source/use-bots.md (hunk 2 above)
Comment (isabelizimm) [reply docs/source/use-bots.md:18]: Works now--this image was committed in a different PR

File: docs/source/use-bots.md (hunk 1 above)
Comment (isabelizimm) [reply docs/source/use-bots.md]: yep-- done!

**Review by chauhankaranraj — APPROVED** [review]
/lgtm :rocket: 
/approve

**Review by oindrillac — COMMENTED** [review]
/lgtm

**Pull request merged**